AsyncDisplayKit/Details/ASCollectionViewFlowLayoutInspector.m

```diff
@@ -79,6 +79,12 @@ - (ASSizeRange)collectionView:(ASCollectionView *)collectionView constrainedSize
   return NodeConstrainedSizeForScrollDirection(collectionView);
 }
 
+- (ASScrollDirection)scrollableDirections
```

Comment:
Yeah, we should definitely pull this out separately: https://github.com/facebook/AsyncDisplayKit/issues/2560